docs/en/stack/ml/df-analytics/ml-dfa-outlier-detection.asciidoc

```diff
@@ -8,28 +8,22 @@
 find unusual data points in a data set compared to the normal data points.
 
 This page contains a conceptual overview of {oldetection} in the {stack} and a 
-simple <<ecommerce-outliers,example>> of how to use it.
+simple <<weblogs-outliers,example>> of how to use it.
 
 
 [discrete]
 [[dfa-outlier-detection]]
 == {oldetection-cap}
```

Comment:
> == {oldetection-cap}

IMO this section title is not required. The "Outlier detection is..." paragraph can replace the "This page..." intro. And if people want to jump right to the example, they can use the links in the lefthand side. If you're thinking of that intro sentence as the "summary" in the frontmatter, I'd stick it in [metadata](https://docs.asciidoctor.org/asciidoc/latest/document/metadata/) for now